sigma9.css

```diff
@@ -7,15 +7,17 @@
 
 @charset "utf-8";
 
+@import url('https://rsms.me/inter/inter.css');
+@import url('https://scp-wiki.wdfiles.com/local--files/component%3Atheme/Sans-Normalcy-v0.1.7.css');
 @import url('https://scp-wiki.wdfiles.com/local--files/component%3Atheme/font-bauhaus.css');
```

Comment:
There should be a comment explaining the continued presence of the Bauhaus font, not just in your PR. This way in the future we don't risk removing it before it's ready.